### PR TITLE
Add tutorial

### PR DESCRIPTION
**Pull request by Izabellaaaq**

### CODE REVIEW

File: how_to_run.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+
+### 1. Create DPE folder and go to the folder:
+ ```
+ mkdir dpe
+ cd dpe
+ ```
+### 2. Setup PUMA Compiler:
+```
+git clone https://izabellaaaq@bitbucket.org/izzat/dpe.git
```
Comment (glaucimar) [inline how_to_run.md]: First, you should not use your user here. and second, let's use the public puma compile repo: https://github.com/illinois-impact/puma-compiler

So, the line should be: git clone https://github.com/illinois-impact/puma-compiler

File: how_to_run.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+```
+# Change here - Specify the Node parameters here
+num_tile_compute = 23 # number of tiles mapped by dnn (leaving input and output tiles) -- (Line 85)
+
+# Do not change this - total number of tiles
+num_tile = num_node * num_tile_compute + 2 # +1 for first tile (I/O tile) - dummy, others - compute -- (Line 95)
+```
+### 7. Run your model, in this example, the ```lstm-layer.cpp```:
+```
+cd /puma-simulator/src
```
Comment (glaucimar) [inline how_to_run.md]: cd "PATH TO PUMA SIMULATOR"/src    ?

File: how_to_run.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+Below you will find some information on how to use the PUMA Compiler and Simulator.
+
+| Requirement | Version                    |
+| ----------- | -------------------------- |
+| OS: Ubuntu  | 16.04.3 LTS (Xenial Xerus) |
+| Python      | 2.7.12      
+
+
+### 1. Create DPE folder and go to the folder:
```
Comment (glaucimar) [inline how_to_run.md]: Create DPE folder and change to the folder:

**Review by glaucimar — CHANGES_REQUESTED** [review]
Thanks Izabella, very good documentation. Please, observe my comments before merging.

**Review by Aayush-Ankit — CHANGES_REQUESTED** [review]
Thanks Izabella. This tutorial is great!
Can we update "dpe" to "puma" in Lines 11, 13 and 14 ?

**Review by glaucimar — APPROVED** [review]
Thanks for addressing comments, Izabella!

**Review by Aayush-Ankit — APPROVED** [review]

**Pull request merged**